gramjs/client/TelegramClient.js

```diff
@@ -1215,7 +1218,9 @@ class TelegramClient {
     }
 
     // endregion
-
 }
 
+// Method mixins
+Object.assign(TelegramClient.prototype, MessageMethods)
```

Comment:
what is this for btw ?

Reply:
This I actually need to remove for this PR, but I'm toying with the idea of using mixins to add methods to `TelegramClient` so that we can have some methods in other files. Similar to the way you're able to do multiple inheritance in Python.